scripts/data_importers/argo.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+
+import pandas as pd
+import xarray as xr
+import defopt
+import seawater
+import os
+import glob
+import data.observational
+from data.observational import Platform, Sample, Station, DataType
+
+VARIABLES = [ 'TEMP', 'PSAL' ]
+
+META_FIELDS = {
+    'PLATFORM_NUMBER': None,
+    'PROJECT_NAME': None,
+    'PI_NAME': None,
+    'DATA_CENTRE': None,
+    'PLATFORM_TYPE': None,
+    'FLOAT_SERIAL_NO': None,
+    'FIRMWARE_VERSION': None,
+    'WMO_INST_TYPE': None
+}
+
+datatype_map = {}
+
+def main(uri: str, filename: str):
+    """Import Argo Profiles
+
+    :param str uri: Database URI
+    :param str filename: Argo NetCDF Filename, or directory of files
+    """
+    data.observational.init_db(uri, echo=False)
+    data.observational.create_tables()
+
+    session = data.observational.db.session
+
+    if os.path.isdir(filename):
+        filenames = sorted(glob.glob(os.path.join(filename, "*.nc")))
```

Comment:
I'm curious about the sorting here...

Reply:
Yeah, that doesn't _need_ to be sorted. I had that in there because I was getting disconnection issues and I wanted to be able to pick up where I left off, but a default glob isn't necessarily sorted.